inst/include/RcppArmadilloAs.h

```diff
@@ -125,7 +127,7 @@ namespace traits {
         typedef typename Rcpp::traits::storage_type<RTYPE>::type value_t;
         Exporter(SEXP x) : vec(x) {}
       
-        cube_t get() {
+        cube_t get(bool copy = true) {
```

Comment:
Should this be `bool copy = false` to shadow the default value Armadillo use (and which you use below) ?